traitsui/qt4/extra/led_editor.py

```diff
@@ -0,0 +1,20 @@
+from pyface.qt import QtGui
```

Comment:
We need a copyright header here.  At a minimum it should be:
```
# (C) Copyright 2019 Enthought, Inc., Austin, TX
# All rights reserved.
```
but we usually also include some text pointing to the license, something like:
```
# This software is provided without warranty under the terms of the BSD
# license included in LICENSE.txt and may be redistributed only
# under the conditions described in the aforementioned license.  The license
# is also available online at http://www.enthought.com/licenses/BSD.txt
# Thanks for using Enthought open source!

Reply:
Added copyright

Reply:
The copyright was only added to the demo file, not this one.